OpenAI.SDK/OpenAiOptions.cs

```diff
@@ -1,9 +1,17 @@
 ﻿namespace OpenAI.GPT3
 {
+    public enum ApiType
+    {
+        OpenAi = 1,
+        Azure = 2
+    }
+
     public class OpenAiOptions
     {
         public static readonly string SettingKey = "OpenAIServiceOptions";
 
+        public virtual ApiType ApiType { get; set; } = ApiType.OpenAi;
```

Comment:
Are there any specific reason to make this property Virtual?

Reply:
Nope, was originally going to make a base class and make the properties readonly, but decided not to and forgot to clean this up.  I'll fix it.

Reply:
please don't change it. I am going to send a commit to fix a couple of things. I am going to include this change too.